svc-bgs-api/src/healthcheck/liveness_script.rb

```diff
@@ -0,0 +1,14 @@
+# This script is used by the K8S Liveness probe to check the connection between RabbitMQ and BGS-api thereby verifying 
+# that the BGS-api application is running and able to perform its basic functions. 
+
+require_relative 'rabbit_subscriber'
+
+def check_rabbitmq_connection
+  subscriber = RabbitSubscriber.new(BUNNY_ARGS)
+  subscriber.connected?
+rescue StandardError => e
+  puts "Liveness check failed: #{e.message}"
+  false
+end
+
+exit check_rabbitmq_connection ? 0 : 1
```

Comment:
I think it helps to have consistency on exists, this vs in svc-bgs-api/src/healthcheck/readiness_script.rb
```
if rabbitmq_connection_active? && bgs_service_available?
  exit 0
else
  puts "Readiness check failed"
  exit 1
end
```

Reply:
Made adjustments to ensure consistency